### ISSUE TITLE
Draw even more icons

### ISSUE THREAD

**Issue by nvkelso** [tangrams/outdoor-style#48]
These aren't in tiles yet, so slipping off into new task.
- [ ] `rest_area` - will be in v1.0 tiles (https://github.com/mapzen/vector-datasource/issues/480)
- [ ] `horse` – for stable and for horse exercise areas, kinda https://github.com/mapzen/vector-datasource/issues/669.
- [ ] `fire_lookout` blocked on https://github.com/mapzen/vector-datasource/issues/726.
- [ ] `fee_station` blocked on https://github.com/mapzen/vector-datasource/issues/726.
